cmd/validate.go

```diff
@@ -66,6 +79,60 @@ this command.
 	},
 }
 
+func validate(ctx context.Context, entity interface{}, kongClient *kong.Client, entityType string) string {
+	_, err := performValidate(ctx, kongClient, entity, entityType)
+	output := ""
+	if err != nil {
+		output = color.New(color.FgRed, color.Bold).Sprintf("%s: %v", entityType, err)
+	}
+	return output
+}
+
+func validateEntities(ctx context.Context, obj interface{}, kongClient *kong.Client, entityType string) {
+	// call GetAll on entity
+	method := reflect.ValueOf(obj).MethodByName("GetAll")
+	entities := method.Call([]reflect.Value{})[0].Interface()
+
+	values := reflect.ValueOf(entities)
+	var wg sync.WaitGroup
+	wg.Add(values.Len())
+	for i := 0; i < values.Len(); i++ {
+		go func(i int) {
+			defer wg.Done()
+			output := validate(ctx, values.Index(i).Interface(), kongClient, entityType)
+			if output != "" {
+				fmt.Println(output)
```

Comment:
Don't print to stdout but instead collect all of these as errors and bubble up all the way and then print them in the RunE fn.

Reply:
I'm not 100% sure my last change is what you were envisioning, but let me know otherwise!